components/Micrio.vue

```diff
@@ -127,19 +178,26 @@ function changeStepBy(delta: number) {
 
 <template>
   <!-- https://kb.micr.io/for-developers/custom-options-for-the-micr-io-element -->
-  <micr-io
-    :id="id"
-    :lang="lang"
-    camspeed="3"
-    controls="false"
-    logo="false"
-    toolbar="false"
-    minimap="false"
-    class="touch-events-none"
-  />
+  <div
+    class="h-full"
+    @touchmove="handleTouchMove"
+  >
+    <micr-io
+      :id="id"
+      :lang="lang"
+      camspeed="3"
+      controls="false"
+      logo="false"
+      toolbar="false"
+      minimap="false"
+    />
+  </div>
   <slot
     :cancelTour="cancelTour"
     :nextMarker="() => changeStepBy(1)"
     :previousMarker="() => changeStepBy(-1)"
+    :isPopoverHidden="isPopoverHidden"
+    :markerRef="markerRef"
+    :currentStepIndex="currentStepIndex"
```

Comment:
This API doesn't really make sense outside of this particular use-case. We're aiming for a re-usable component here which we can later deploy in other apps.

Could `currentStep` and `isPopoverHidden` be devised from `marker`?

Something like this would be nicer (but it might need some work to make sure it conveys everything it should convey):

```suggestion
    :marker="markerRef"
```